--- setup.py
+++ setup.py
@@ -26,8 +26,8 @@
     # win32 APIs if on Windows (required for npipe support)
     # Python 3.6 is only compatible with v220 ; Python < 3.5 is not supported
     # on v220 ; ALL versions are broken for v222 (as of 2018-01-26)
-    ':sys_platform == "win32" and python_version < "3.6"': 'pypiwin32==219',
-    ':sys_platform == "win32" and python_version >= "3.6"': 'pypiwin32==223',
+    ':sys_platform == "win32" and python_version < "3.6"': 'pywin32==219',
+    ':sys_platform == "win32" and python_version >= "3.6"': 'pywin32==223',
 
     # If using docker-py over TLS, highly recommend this option is
     # pip-installed or pinned.
